docs/sources/panels/panel-library.md

```diff
@@ -0,0 +1,75 @@
++++
+title = "Library panels"
+weight = 400
++++
+
+# Library panel
+
+Library panels allow users to create reusable panels that aren’t tied to any one dashboard. As a result, any changes made to one instance of the library panel affecting all other instances.
+
+## Create a library panel
+
+Before you can use library panels, you have to create them.
+
+To create a library panel:
+
+1. Create a panel as you normally would, following instructions in [Add a panel]({{< relref "./add-a-panel.md" >}}).
```

Comment:
How about:

1. Pick an existing panel or create a panel as you normally would, following instructions in [Add a panel]({{< relref "./add-a-panel.md" >}}).

I think that for many of our community members, they'll be approaching this from an existing panel perspective.

Reply:
Add a panel is really creating a panel. I will modify this a little - to make it clear that you can use an existing panel as well.